onto-viewer-web-app/pom.xml

```diff
@@ -72,6 +72,13 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
```

Comment:
Use `org.junit.jupiter:junit-jupiter-engine` instead. As its version is specified in the parent POM, you don't have to specify it explicitly. If you use the `jar` type, don't specify it as it the default one.